encoder/audio.py

```diff
@@ -5,10 +5,12 @@
 import numpy as np
 import librosa
 import struct
+import warnings
 
 try:
     import webrtcvad
 except:
+    print("WARNING: Unable to import 'webrtcvad'. Please install for better noise removal.")
```

Comment:
```suggestion
    warn("Unable to import 'webrtcvad'. This package enables noise removal and is recommended.")
```

Reply:
This latest commit implements your suggestions. Let me know if you think of anything else